frontend/src/components/ibutsu-header.js

```diff
@@ -227,8 +232,11 @@ export class IbutsuHeader extends React.Component {
       return;
     }
     // update context
-    setPrimaryObject(value)
-    setPrimaryType('project')
+    setPrimaryObject(value);
+    setPrimaryType('project');
+    console.log('project selected: ' + value.default_dashboard_id)
```

Comment:
rogue console.

Reply:
thx, forgot to drop all these before marking ready.